packages/exchange-analytics/src/config/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Requester } from '@chainlink/ea-bootstrap'
```

Comment:
https://github.com/linkpoolio/adapters/tree/develop/packages/kyc/src/config

IMHO the config I used for `kyc-adapter` is more consistent with our previous experience handling the env vars in `Config` and/or and extended `Config`. And now centralises multiple DP configs in one place.

As none of the 3 DPs require API keys you'll have something like:
```typescript
export type BinanceApiProviderConfig = ApiProviderConfig
export type BybitApiProviderConfig = ApiProviderConfig
export type FtxApiProviderConfig = ApiProviderConfig

export type SupportedApiProviderConfig = BinanceApiProviderConfig | BybitApiProviderConfig | FtxApiProviderConfig
```